packages/wdio-reporter/src/stats/runner.ts

```diff
@@ -0,0 +1,44 @@
+import RunnableStats, { Runnable } from './runnable'
+import { sanitizeCaps } from '../utils'
+import { DesiredCapabilities, Options } from '../types'
+
+export interface Runner extends Runnable {
+    cid: string
+    capabilities: DesiredCapabilities
+    config: Options
+    specs: string[]
+    sessionId: string
+    isMultiremote: boolean
+    retry: number
+    failures: number
+    retries: number
+}
+
+/**
+ * Class to capture statistics about a test run. A test run is a single instance that
+ * runs one or more spec files
+ */
+export default class RunnerStats extends RunnableStats {
+    cid: string
+    capabilities: DesiredCapabilities
+    sanitizedCapabilities: string
+    config: Options
+    specs: string[]
+    sessionId: string
+    isMultiremote: boolean
+    retry: number
+    failures?: number
```

Comment:
Should this be undefined, or instantiate with 0? Same for retries on next line.

Reply:
can you take a look at how it used to work before?

Reply:
Well, the current behaviour is what it was. I just thought it might make more sense to have it at 0. But 🤷‍♂️